### ISSUE TITLE
More than 21 sec to recieve green-red buttons of echo test

### ISSUE THREAD

**Issue by bendup99** [bigbluebutton/bigbluebutton#9095]
Hi there, 

First of all a warm congrat to all the bbb team for your great job in this platform !

I installed 3 new bbb servers, and we always have same issue : there is a delay of at leat 21 sec to recieve the green-red buttons of the echo test.

Everything is working well, excepts this strange behavior.

Some times, it take up to 40 secs to recieve the G/R buttons.

Any idea to tune an/or fix this ?

Thanks in dvance,
Ben

**Comment by basisbit** [comment]
To analyze this issue, we need more information about your server configuration, bbb server version, what stun / turn server you are using (the default freeswitch stun servers seem to be slow to respond lately), webbrowser type + version and also the console log from your webbrowsers developer tools.

Also, could you please help us test the recent change #9084 by applying it to your bbb server configuration, and then executing `bbb-conf --restart` to restart the BigBlueButton services?

**Comment by basisbit** [comment]
@bendup99 any update on this?

**Comment by bendup99** [comment]
Hi, here are infos about this 

bbb : latest downloaded end march 2020
webbrowsser : Firefox 74 (same behavior wirth FF 75)
bbb server version : 2.5.2
bbb client version : 862
server components : everything installed according to bbb procedure
server installed early april 2020


Here are the FF console logs during those 20 seconds betweeb click on the mix icon and recieving green/red butons 


[10:29:14:0738] INFO: clientLogger: User requested to join audio conference with mic https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1096863 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
[10:29:14:0801] DEBUG: clientLogger: Creating the user agent https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1126740 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | configuration parameters after validation: sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · viaHost: "46l1j8o2pbg5.invalid" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · uri: sip:w_r5pr2zbyhfgl_2-bbbID-USER@visio.XXXX.YYY sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · wsServers: [{"ws_uri":"wss://visio.XXXX.YYY/ws","sip_uri":"<sip:visio.XXXX.YYY;transport=ws;lr>","weight":0,"status":0,"scheme":"WSS"}] sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · custom: {} sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · displayName: "w_r5pr2zbyhfgl_2-bbbID-USER" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · password: NOT SHOWN sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · registerExpires: 600 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · register: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · registrarServer: sip:visio.XXXX.YYY sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · wsServerMaxReconnection: 3 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · wsServerReconnectionTimeout: 4 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · connectionRecoveryMinInterval: 2 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · connectionRecoveryMaxInterval: 30 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · keepAliveInterval: 0 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · extraSupported: [] sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · usePreloadedRoute: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · userAgentString: "BigBlueButton" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · iceCheckingTimeout: 5000 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · noAnswerTimeout: 60000 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · stunServers: ["stun:stun.freeswitch.org"] sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · turnServers: [] sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · traceSip: true sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackViaTcp: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackIpInContact: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackWssInTransport: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackAllowUnregisteredOptionTags: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackCleanJitsiSdpImageattr: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackStripTcp: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackPlanBUnifiedPlanTranslation: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hackAddAudioTransceiver: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · relayOnlyOnReconnect: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · contactTransport: "ws" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · forceRport: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · autostart: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · autostop: true sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · rel100: "none" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · replaces: "none" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · mediaHandlerFactory: function defaultFactory (session, options) { return new MediaHandler(session, options); } sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · authenticationFactory: undefined sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · allowLegacyNotifications: false sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · localSdpCallback: undefined sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · remoteSdpCallback: undefined sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · instanceId: "969e382d-0e60-44c7-bfde-f6f284439874" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · sipjsId: "9e65o" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · hostportParams: "visio.XXXX.YYY" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · authorizationUser: "w_r5pr2zbyhfgl_2-bbbID-USER" sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | · media: undefined sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | user requested startup... sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.transport | connecting to WebSocket wss://visio.XXXX.YYY/ws sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.transport | WebSocket wss://visio.XXXX.YYY/ws connected sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.ua | connection state set to 0 sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | acquiring local media sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | acquired local media streams sip.js:2900:10
onaddstream is deprecated! Use peerConnection.ontrack instead. sip.js:11518
RTCPeerConnection.getLocalStreams/getRemoteStreams sont obsolètes. Veuillez utiliser RTCPeerConnection.getSenders/getReceivers à la place. sip.js:11419:37
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | RTCIceGatheringState changed: gathering sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:0 1 UDP 2122187007 90.90.90.1 52687 typ host sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2 1 UDP 2122121471 192.168.0.159 52688 typ host sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:4 1 UDP 2122252543 172.18.59.225 52689 typ host sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:6 1 TCP 2105458943 90.90.90.1 9 typ host tcptype active sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:7 1 TCP 2105393407 192.168.0.159 9 typ host tcptype active sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:8 1 TCP 2105524479 172.18.59.225 9 typ host tcptype active sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:0 2 UDP 2122187006 90.90.90.1 52690 typ host sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:2 2 UDP 2122121470 192.168.0.159 52691 typ host sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:4 2 UDP 2122252542 172.18.59.225 52692 typ host sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:6 2 TCP 2105458942 90.90.90.1 9 typ host tcptype active sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:7 2 TCP 2105393406 192.168.0.159 9 typ host tcptype active sip.js:2900:10
Mon Apr 20 2020 10:29:14 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:8 2 TCP 2105524478 172.18.59.225 9 typ host tcptype active sip.js:2900:10
Mon Apr 20 2020 10:29:15 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:3 1 UDP 1685921791 82.226.255.15 52688 typ srflx raddr 192.168.0.159 rport 52688 sip.js:2900:10
Mon Apr 20 2020 10:29:15 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | ICE candidate received: candidate:3 2 UDP 1685921790 82.226.255.15 52691 typ srflx raddr 192.168.0.159 rport 52691 sip.js:2900:10
Mon Apr 20 2020 10:29:19 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | RTCIceChecking Timeout Triggered after 5000 milliseconds sip.js:2900:10
[10:29:20:0003] INFO: clientLogger: SDP looks like Unified Plan https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1164996 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
[10:29:20:0008] INFO: clientLogger: Found candidates with type v4 (public? true) and without type v6 https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1167147 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
[10:29:20:0010] INFO: clientLogger: Found a server reflexive candidate https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1167439 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
[10:29:20:0012] INFO: clientLogger: No peer reflexive candidate found https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1167827 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
[10:29:20:0013] INFO: clientLogger: No relay candidate found https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1168045 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
Mon Apr 20 2020 10:29:20 GMT+0200 (heure d’été d’Europe centrale) | sip.transport | sending WebSocket message: INVITE sip:919639121@visio.XXXX.YYY SIP/2.0 Via: SIP/2.0/WSS 46l1j8o2pbg5.invalid;branch=z9hG4bK7474565 Max-Forwards: 70 To: <sip:919639121@visio.XXXX.YYY> From: "w_r5pr2zbyhfgl_2-bbbID-USER" <sip:w_r5pr2zbyhfgl_2-bbbID-USER@visio.XXXX.YYY>;tag=5b077fuu53 Call-ID: 9e65o9lvhnhv9qnd9m86 CSeq: 518 INVITE Contact: <sip:2162qmhu@46l1j8o2pbg5.invalid;transport=ws;ob> Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER Supported: outbound User-Agent: BigBlueButton Content-Type: application/sdp Content-Length: 2004 v=0 o=mozilla...THIS_IS_SDPARTA-75.0 6438386851153934757 0 IN IP4 0.0.0.0 s=- t=0 0 a=sendrecv a=fingerprint:sha-256 9B:87:AD:EA:93:88:93:48:BD:6F:E3:90:19:02:0B:C5:02:FC:9E:2D:90:70:23:50:49:1E:E8:53:F5:28:20:C9 a=ice-options:trickle a=msid-semantic: WMS * a=group:BUNDLE 0 m=audio 52688 UDP/TLS/RTP/SAVPF 109 9 0 8 101 c=IN IP4 82.226.255.15 a=rtpmap:109 opus/48000/2 a=rtpmap:9 G722/8000/1 a=rtpmap:0 PCMU/8000 a=rtpmap:8 PCMA/8000 a=rtpmap:101 telephone-event/8000 a=fmtp:109 maxplaybackrate=48000;stereo=1;useinbandfec=1 a=fmtp:101 0-15 a=rtcp:52691 IN IP4 82.226.255.15 a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level a=extmap:2/recvonly urn:ietf:params:rtp-hdrext:csrc-audio-level a=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid a=setup:actpass a=mid:0 a=msid:{4d443727-5c59-4bc4-9329-aa74ff2b913d} {70dc7acb-7cf7-4eef-9c23-dc0d5770e306} a=sendrecv a=ice-ufrag:a0371112 a=ice-pwd:4eecc23b9bf6a2fc3d243125f4ce0b85 a=candidate:0 1 UDP 2122187007 90.90.90.1 52687 typ host a=candidate:2 1 UDP 2122121471 192.168.0.159 52688 typ host a=candidate:4 1 UDP 2122252543 172.18.59.225 52689 typ host a=candidate:6 1 TCP 2105458943 90.90.90.1 9 typ host tcptype active a=candidate:7 1 TCP 2105393407 192.168.0.159 9 typ host tcptype active a=candidate:8 1 TCP 2105524479 172.18.59.225 9 typ host tcptype active a=candidate:0 2 UDP 2122187006 90.90.90.1 52690 typ host a=candidate:2 2 UDP 2122121470 192.168.0.159 52691 typ host a=candidate:4 2 UDP 2122252542 172.18.59.225 52692 typ host a=candidate:6 2 TCP 2105458942 90.90.90.1 9 typ host tcptype active a=candidate:7 2 TCP 2105393406 192.168.0.159 9 typ host tcptype active a=candidate:8 2 TCP 2105524478 172.18.59.225 9 typ host tcptype active a=candidate:3 1 UDP 1685921791 82.226.255.15 52688 typ srflx raddr 192.168.0.159 rport 52688 a=candidate:3 2 UDP 1685921790 82.226.255.15 52691 typ srflx raddr 192.168.0.159 rport 52691 a=ssrc:2779308677 cname:{bf3f2c92-c6db-4939-b572-aff0e5e396cc} a=rtcp-mux sip.js:2900:10
Mon Apr 20 2020 10:29:20 GMT+0200 (heure d’été d’Europe centrale) | sip.transport | received WebSocket text message: SIP/2.0 100 Trying Via: SIP/2.0/WSS 46l1j8o2pbg5.invalid;branch=z9hG4bK7474565;received=93.184.35.186;rport=57324 From: "w_r5pr2zbyhfgl_2-bbbID-USER" <sip:w_r5pr2zbyhfgl_2-bbbID-USER@visio.XXXX.YYY>;tag=5b077fuu53 To: <sip:919639121@visio.XXXX.YYY> Call-ID: 9e65o9lvhnhv9qnd9m86 CSeq: 518 INVITE User-Agent: FreeSWITCH-mod_sofia/1.10.3-dev+git~20200122T195942Z~6d8e8f35b4~64bit Content-Length: 0 sip.js:2900:10
[10:29:20:0096] INFO: clientLogger: Audio call session progress update https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1129300 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
SIP.EventEmitter#off is deprecated and may be removed in future SIP.js versions. Please use removeListener or removeAllListeners instead. See here for more details: http://nodejs.org/api/events.html#events_emitter_removelistener_event_listener sip.js:1272:11
Mon Apr 20 2020 10:29:20 GMT+0200 (heure d’été d’Europe centrale) | sip.transport | received WebSocket text message: SIP/2.0 200 OK Via: SIP/2.0/WSS 46l1j8o2pbg5.invalid;branch=z9hG4bK7474565;received=93.184.35.186;rport=57324 From: "w_r5pr2zbyhfgl_2-bbbID-USER" <sip:w_r5pr2zbyhfgl_2-bbbID-USER@visio.XXXX.YYY>;tag=5b077fuu53 To: <sip:919639121@visio.XXXX.YYY>;tag=QcQj7eFZB7QeF Call-ID: 9e65o9lvhnhv9qnd9m86 CSeq: 518 INVITE Contact: <sip:919639121@93.184.35.186:5060;transport=udp> User-Agent: FreeSWITCH-mod_sofia/1.10.3-dev+git~20200122T195942Z~6d8e8f35b4~64bit Accept: application/sdp Allow: INVITE, ACK, BYE, CANCEL, OPTIONS, MESSAGE, INFO, UPDATE, REGISTER, REFER, NOTIFY Supported: timer, path, replaces Allow-Events: talk, hold, conference, refer Content-Type: application/sdp Content-Disposition: session Content-Length: 884 Remote-Party-ID: "919639121" <sip:919639121@visio.XXXX.YYY>;party=calling;privacy=off;screen=no v=0 o=FreeSWITCH 1587351191 1587351192 IN IP4 93.184.35.186 s=FreeSWITCH c=IN IP4 93.184.35.186 t=0 0 a=msid-semantic: WMS AVHnhVFiXdc3ZHfJ2MFEZZTnzvRO48Du m=audio 20168 UDP/TLS/RTP/SAVPF 109 101 a=rtpmap:109 opus/48000/2 a=fmtp:109 useinbandfec=1; stereo=1 a=rtpmap:101 telephone-event/8000 a=ptime:20 a=fingerprint:sha-256 40:0B:13:3C:08:BF:90:55:59:40:FC:35:5B:28:7D:9A:62:24:54:27:E2:27:F1:AC:F4:90:97:99:3E:1D:96:9F a=setup:active a=rtcp-mux a=rtcp:20168 IN IP4 93.184.35.186 a=ice-ufrag:la6aZPnpb5353xpN a=ice-pwd:yXEl0DU7Fn3eWyF7zC8phmEi a=candidate:3960913721 1 udp 659136 93.184.35.186 20168 typ host generation 0 a=end-of-candidates a=s²src:1990503654 cname:TbH1Wd9JgbD5IIFB a=ssrc:1990503654 msid:AVHnhVFiXdc3ZHfJ2MFEZZTnzvRO48Du a0 a=ssrc:1990503654 mslabel:AVHnhVFiXdc3ZHfJ2MFEZZTnzvRO48Du a=ssrc:1990503654 label:AVHnhVFiXdc3ZHfJ2MFEZZTnzvRO48Dua0 sip.js:2900:10
Mon Apr 20 2020 10:29:20 GMT+0200 (heure d’été d’Europe centrale) | sip.dialog | new UAC dialog created with status CONFIRMED sip.js:2900:10
Mon Apr 20 2020 10:29:20 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | stream added: AVHnhVFiXdc3ZHfJ2MFEZZTnzvRO48Du sip.js:2900:10
Mon Apr 20 2020 10:29:20 GMT+0200 (heure d’été d’Europe centrale) | sip.transport | sending WebSocket message: ACK sip:919639121@93.184.35.186:5060;transport=udp SIP/2.0 Via: SIP/2.0/WSS 46l1j8o2pbg5.invalid;branch=z9hG4bK7564917 Max-Forwards: 70 To: <sip:919639121@visio.XXXX.YYY>;tag=QcQj7eFZB7QeF From: "w_r5pr2zbyhfgl_2-bbbID-USER" <sip:w_r5pr2zbyhfgl_2-bbbID-USER@visio.XXXX.YYY>;tag=5b077fuu53 Call-ID: 9e65o9lvhnhv9qnd9m86 CSeq: 518 ACK Supported: outbound User-Agent: BigBlueButton Content-Length: 0 sip.js:2900:10
[10:29:20:0121] INFO: clientLogger: Audio call session accepted https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1128989 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
SIP.EventEmitter#off is deprecated and may be removed in future SIP.js versions. Please use removeListener or removeAllListeners instead. See here for more details: http://nodejs.org/api/events.html#events_emitter_removelistener_event_listener sip.js:1272:11
[10:29:20:0123] INFO: clientLogger: The audio element played successfully https://visio.XXXX.YYY/html5client/compatibility/sip.js?v=862:11782:35 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
[10:29:20:0191] INFO: clientLogger: ICE connection success. Current state - connected https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1129449 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
SIP.EventEmitter#off is deprecated and may be removed in future SIP.js versions. Please use removeListener or removeAllListeners instead. See here for more details: http://nodejs.org/api/events.html#events_emitter_removelistener_event_listener 2 sip.js:1272:11
[10:29:20:0196] INFO: clientLogger: ICE Candidate selected - type: srflx, protocol: udp, ipProtocol: v4 https://visio.XXXX.YYY/html5client/e438134a1dbc31c93d665b22433904df4108a414.js?meteor_js_resource=true:121:1168387 e438134a1dbc31c93d665b22433904df4108a414.js:9:342209
Mon Apr 20 2020 10:29:25 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | RTCIceChecking Timeout Triggered after 5000 milliseconds sip.js:2900:10
Mon Apr 20 2020 10:29:26 GMT+0200 (heure d’été d’Europe centrale) | sip.invitecontext.mediahandler | RTCIceGatheringState changed: complete

**Comment by basisbit** [comment]
Thank you for providing the log. From what it looks like, the used webbrowser is Google Chrome, and of the ~12 seconds of "connecting" time a total of 10 seconds is spent for just waiting on the IceCheckingTimeout to happen.
The already mentioned pull request (improvement pending for bbb version 2.3) should fix your problem by reducing that 5 seconds IceCheckingTimeout to 1 second.
Could you please help us test this change by applying it (#9084) to your bbb server configuration, and then executing bbb-conf --restart to restart the BigBlueButton services. After applying this change, ask your users if they all still can connect without a problem and please report the results here.

**Comment by nugzarg** [comment]
The same issue under manjaro linux and firefox 75. Echo test takes 30-32 seconds. With chrome echo test requires about 5-7 seconds. It seems that only linux version of firefox is affected. Under windows 10 firefox 75 takes only 2-3 seconds for echo test.

**Comment by basisbit** [comment]
> Echo test takes 30-32 seconds

Could you please provide the firefox developer tools (press F12 key to open) console output for those 30 seconds? You can use https://gist.github.com to paste the log and then link to in in your answer here. Thank you!

**Comment by nugzarg** [comment]
Here is the log from firefox 75 under manjaro linux:
https://gist.github.com/nugzarg/eadcc327fbf161b8ba4df5faeeda07cc

**Comment by matiasilva** [comment]
No follow up in 2 months. Please feel free to post further support issues in https://groups.google.com/forum/#!forum/bigbluebutton-setup. Thanks

**Comment by paulmenzel** [comment]
I’d say, this is a duplicate of https://github.com/bigbluebutton/bigbluebutton/issues/8824.
